config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :companies
```

Comment:
move to account namespace

Reply:
Added account namespace and refactored routes, controller, views and tests with new path helpers